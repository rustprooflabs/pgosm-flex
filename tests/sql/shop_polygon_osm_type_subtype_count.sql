SELECT osm_type COLLATE "C", osm_subtype COLLATE "C", COUNT(*)
    FROM osm.shop_polygon
    GROUP BY osm_type COLLATE "C", osm_subtype COLLATE "C"
    ORDER BY osm_type COLLATE "C", osm_subtype COLLATE "C"
;