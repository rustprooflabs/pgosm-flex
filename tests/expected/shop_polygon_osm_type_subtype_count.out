amenity|cafe|10
amenity|car_rental|1
amenity|pharmacy|8
amenity|pub|5
shop|alcohol|15
shop|art|1
shop|art;paper|1
shop|bakery|3
shop|beauty|7
shop|bicycle|1
shop|books|1
shop|car|4
shop|car_parts|3
shop|car_repair|7
shop|chemist|2
shop|clothes|8
shop|convenience|35
shop|copyshop|1
shop|cosmetics|2
shop|deli|1
shop|department_store|5
shop|doityourself|2
shop|dry_cleaning|4
shop|electronics|1
shop|florist|1
shop|frame|2
shop|funeral_directors|4
shop|garden_centre|1
shop|gift|5
shop|greengrocer|1
shop|hairdresser|10
shop|interior_decoration|2
shop|laundry|5
shop|mall|3
shop|mobile_phone|1
shop|paint|1
shop|pet|1
shop|religion|1
shop|seafood|5
shop|shoes|2
shop|sports|2
shop|supermarket|15
shop|ticket|1
shop|tobacco|2
shop|tyres|1
shop|vacant|8
shop|variety_store|3
shop|wholesale|1
shop|yes|1
